namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/pic-probation-offender-events-sub-queue.tf

```
@@ -99,10 +99,10 @@ resource "kubernetes_secret" "pic_probation_offender_events_dead_letter_queue" {
   }
 }
 
-resource "kubernetes_secret" "pic_probation_sqs_tool_main_queue" {
+resource "kubernetes_secret" "pic_probation_offender_events_sqs_tool_main_queue" {
   metadata {
-    name      = "pic-probation-sqs-tool-main-queue"
-    namespace = "pic-dev"
+    name      = "pic-probation-offender-events-sqs-tool-main-queue"
```

This is the name of the secret as it will appear in our k8s namespace so the `pic` prefix is redundant as that whole namespace belongs to us.

```suggestion
    name      = "probation-offender-events-court-case-service-main-queue"
```